packages/gatsby-admin/src/components/providers.tsx

```diff
@@ -1,7 +1,63 @@
 import React from "react"
-import { Provider } from "urql"
-import client from "../urql-client"
+import { Provider, Client } from "urql"
+import { ThemeProvider, getTheme } from "gatsby-interface"
+import { ThemeProvider as StrictUIProvider } from "strict-ui"
+import { createUrqlClient } from "../urql-client"
 
-export default ({ children }): React.ReactElement => (
-  <Provider value={client}>{children}</Provider>
+const baseTheme = getTheme()
+
+const theme = {
+  ...baseTheme,
+  colors: {
+    ...baseTheme.colors,
+    background: baseTheme.colors.grey[90],
+  },
+  fontWeights: {
+    ...baseTheme.fontWeights,
+    "500": 500,
+  },
+  borders: {
+    default: `1px solid ${baseTheme.colors.whiteFade[20]}`,
+  },
+  sizes: {
+    // NOTE(@mxstbr): Hacks around issues with strict-ui that I have to fix upstream eventually
```

Comment:
It's the same thing :wink: strict-ui is literally theme-ui, but it throws an error if you try to use a non-theme value.

You can drop down to theme-ui in any file, which I've actually done in some, by simply doing `/** @jsx jsx */ import { jsx } from 'theme-ui'`. Completely interoperable!

Reply:
I don't think it's fair to say they are the same, since Theme UI works more as a *superset* of CSS and strict-mode is inherently a subset of that API, which means some things would not work as expected for people coming from using Theme UI -- that said, I don't think this should block the initial release, but we should be considering what the "SDK" for Admin will look like and how easy it is for others to work on Admin

Reply:
My answer of "they are the same thing" was that strict-ui literally uses theme-ui under the hood to alleviate the mentioned concern that theme-ui will get new features but strict-ui will not, which cannot be the case. Sorry for not being clear!

To clarify, a pseudo-implementation of strict-ui would look like this:

```JS
const jsx = (...args) => {
  strictMode(...args)
  return themeUiJSX(...args)
}
```

It is the theme-ui _API_, with some additional validation on top.

Reply:
i think the most questions arising, because the [npm package readme](https://www.npmjs.com/package/strict-ui) is empty and there is no github repo associated :/